internal/api/token.go

```diff
@@ -48,6 +49,7 @@ func (r *AccessTokenResponse) AsRedirectURL(redirectURL string, extraParams url.
 	extraParams.Set("access_token", r.Token)
 	extraParams.Set("token_type", r.TokenType)
 	extraParams.Set("expires_in", strconv.Itoa(r.ExpiresIn))
+	extraParams.Set("expires_at", strconv.FormatInt(r.ExpiresAt, 10))
```

Comment:
Can consider using an `int` and also using `Itoa`  like with `expires_in` if there are no special considerations around using an `int64` represenation

Reply:
`int` may be too small to fit seconds in the future.